mail/testing/src/main/java/com/fsck/k9/mail/testing/StringHelper.kt

```diff
@@ -1,4 +1,4 @@
-package com.fsck.k9.mail
+package com.fsck.k9.mail.testing
```

Comment:
Does this need to go in a non-java folder?

Reply:
Same for a few other files

Reply:
The java source folder could host Java and Kotlin files. So as long there are still some Java files present it's not strictly necessary to separate them. The Kotlin folder on the other side can't contain Java files.

Using the kotlin source set prevents from accidentally adding Java files and the compiler could work a little bit more efficient.